src/api/repository.py

```diff
@@ -151,48 +152,84 @@ def __init__(self, session):
         super().__init__(session, models.Vote)
 
 
+class SqlAlchemyPoliticianRepository(SqlAlchemyBaseRepository):
+    def __init__(self, session):
+        super().__init__(session, models.Politician)
+
+
 class SqlAlchemyFactory:
     """Factory class to create repositories"""
 
     def __init__(self, session):
         self.session = session
+        """Must add new repositories to the map to be able to create them"""
+        self.entity_factory_map = {
+            Entity.PARTY.value: self.create_party_repository,
+            Entity.PARTY_STYLE.value: self.create_party_style_repository,
+            Entity.POLITICIAN.value: self.create_politician_repository,
```

Comment:
I added the Politician entity and function pair.

Please ensure that you add it to the next one.

If it does not make sense, please look at the comment on utils.py(get_repository method).